### DISCUSSION TITLE
Playwright integration and glsp-vscode-integration - Integrated Server

### DISCUSSION THREAD

**Discussion by jmenzies12** [eclipse-glsp/glsp#1396] [Q&A]
Hey there,

I am trying to get the playwright integration working with a modified version of the task and workflow examples. We are using the integrated server solution ie NodeGlspVscodeServer.

When I run a test it successfully installs the extension but then it loads up VSCode and sits on a blank screen until inevitably timing out. I was wondering if it doesn't work with the integrated server? Or if I have to customize some config in order to get it working?

Kind Regards
Jonathan

**Comment by tortmayr** [comment]
Hi @jmenzies12,

the playwright integration should work perfectly fine when using an integrated node server. There are no known issues or
additional configuration required. I quickly tested this adopting the workflow example to use the integrated server as default and everything works as expected.

Before we dive further into this: Did you make sure that the generated .vsix file is working as expected?
Does the GLSP diagram editor show up if you install the extension manually and open a corresponding file?

**Reply by jmenzies12** [reply]
Hey!

Yes our .vsix works as expected when installed in vscode or in theia.
